Patching to get the .so version the same as in the X gate

--- a/src/Makefile.am	2018-01-16 07:49:36.670070924 -0800
+++ b/src/Makefile.am	2018-01-16 07:49:47.825022138 -0800
@@ -65,4 +65,4 @@
 
 libXfont_la_SOURCES = dummy.c
 
-libXfont_la_LDFLAGS = -version-number 1:4:1 -no-undefined
+libXfont_la_LDFLAGS = -version-number 1 -no-undefined
